syntax = "proto3";

package text.classification.proto.grpc;


// classifies string and returns a class name as a response
service TextClassificationService {
    rpc Classify (InputText) returns (Prediction) {}
}

message InputText {
    string intput_text = 1;
}

message Prediction {
    string class_name = 1;
    float confidence = 2;
}